modules/apps/portal-template/portal-template-react-renderer-impl/src/main/resources/META-INF/resources/index.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+// render API
```

Comment:
Kill this comment?

Reply:
I think we add those comments in `index.js` files (I copied it from the original location). They also show up [here](https://github.com/liferay/liferay-portal/blob/bd34cdc5b3886ccdfb4d73f1fd5423ff9e5567b0/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/index.es.js#L22-L21).

Reply:
The comments in the index you linked seem fine (kind of like section headers), but here they obviously add nothing.

Reply:
A pattern is a pattern :sweat_smile:

Reply:
![riiiight](https://user-images.githubusercontent.com/7074/101883299-58bfd700-3b97-11eb-8cdd-7b326a75ccdb.gif)

Reply:
> A pattern is a pattern 😅

It's not a pattern when there are more examples pointing to the contrary.

We only added the comments in `frontend-js-web` to give it some order and avoid getting a long list of confusing alphabetically-sorted but semantically-unsorted exports